pubspec.yaml

```diff
@@ -111,31 +111,35 @@ dependencies:
   matrix_homeserver_recommendations: ^0.3.0
   matrix_link_text: ^2.0.0
   native_imaging: ^0.1.0
-  package_info_plus: ^4.0.0
+  package_info_plus: ^8.0.0
   path_provider: ^2.0.15
-  permission_handler: ^10.4.5
+  permission_handler: ^11.0.1
   pin_code_text_field: ^1.8.0
   provider: ^6.0.2
   punycode: ^1.0.0
   qr_code_scanner: ^1.0.0
   qr_flutter: ^4.0.0
   record: ^4.4.4
   scroll_to_index: ^3.0.1
-  share_plus: ^4.0.10+1
-  shared_preferences: 2.0.15 # Pinned because https://github.com/flutter/flutter/issues/118401
```

Comment:
please demo this case

Reply:
https://github.com/linagora/twake-on-matrix/assets/43041967/18f49848-84c1-4569-914d-cdc72d3a8883

Reply:
https://github.com/linagora/twake-on-matrix/assets/43041967/13eaed30-7776-4769-bbca-47082e7b0ab9